Tests/Realm.Tests/Sync/SyncConfigurationTests.cs

```diff
@@ -30,8 +30,7 @@ public class SyncConfigurationTests : SyncTestBase
         [Test]
         public void SyncConfiguration_WithoutPath()
         {
-            var user = GetFakeUser();
```

Comment:
The changes here are somewhat unrelated - I restructured the `SyncTestBase` a little and those were caught up in the refactoring.